src/compression.rs

```diff
@@ -91,6 +96,7 @@ impl CompressionMethod {
             8 => CompressionMethod::Deflated,
             #[cfg(feature = "bzip2")]
             12 => CompressionMethod::Bzip2,
+            99 => CompressionMethod::Aes,
```

Comment:
Should we mode it under the feature?

Reply:
Having the `Aes` variant outside the feature is intended. It doesn't require any dependencies and makes it possible to recognize encrypted files and give a more fitting error message, see [here](https://github.com/zip-rs/zip/pull/203/files#diff-f5613ff66dd29a0567e4173ff7c04ed28b08eb7384040946dd410698700b87eeR213-R218).

Reply:
Same problem as #266. We'll have to change `CompressionMethod::Aes` to `CompressionMethod::AES`, but I dont mind doing that myself (sadly I can't edit the PR :D)

Reply:
The constant has been added, but not sure if it's how you want it to be used.